docs/layers/introduction.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Introduction
+sidebar_label: Introduction
+slug: /layers
+---
+
+# What are Layers?
+Layers are a way of performing experiments with out requring continued updates to your codebase. They are a 'write once' solution to experimentation. Layers work with Experiments to run tests on product features, and allow shared parameters between Experiments.
+
+## When to use Layers
+Similar to our Experiments product, Layers are ideal for running A/B or A/B/n experiments. There differentiator is that you can avoid rewritting code between experiment runs.
+
+
+## Layers vs Experiments 
+Why would you want to use Layers? Let's say you have an experiment that changes the color of the checkout button in your purchase flow. This code might look something like this:
```

Comment:
This isnt even a question.  You need an experiment either way, a layer does not run a test without an experiment

Reply:
I think it is fair to ask "Why would you want to *add layers* to your experiments?" or "run your experiments in layers" or something, but agreed we shouldn't make it sound like it replaces Experiments